### PR TITLE
add: `an-anime-game-launcher-gtk-bin`

### PR DESCRIPTION
**Pull request by Mar0xy**
Due to debian missing libadwaita-1-0 currently It will error out for debian users also causing the CI to fail.

### CODE REVIEW

File: packages/an-anime-game-launcher-gtk-bin/an-anime-game-launcher-gtk-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,54 @@
+if [[ "$(grep -Po 'ID=\K[^"]+' /etc/os-release)" == "debian" ]]; then
+    fancy_message error "This script doesn't work on Debian"
+    cleanup # Trigger pacstall cleanups
+    exit 1
+fi
+
+name=an-anime-game-launcher-gtk-bin
+pkgname=an-anime-game-launcher
+version=1.0.0
```
Comment (Henryws) [inline packages/an-anime-game-launcher-gtk-bin/an-anime-game-launcher-gtk-bin.pacscript:9]: Quote these.

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/an-anime-game-launcher-gtk-bin/an-anime-game-launcher-gtk-bin.pacscript (hunk 1 above)
Comment (Mar0xy) [reply packages/an-anime-game-launcher-gtk-bin/an-anime-game-launcher-gtk-bin.pacscript]: Done

File: packages/an-anime-game-launcher-gtk-bin/an-anime-game-launcher-gtk-bin.pacscript [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+depends="libayatana-appindicator3-1 tar unzip xdelta3 cabextract git libadwaita-1-0"
+
+optdepends=(
+    "mangohud: FPS Hud/GUI"
+    "gamemode: Game Optimizations"
+)
+
+prepare() {
+    sudo chmod +x "${SRCDIR}/an-anime-game-launcher"
```
Comment (Henryws) [inline packages/an-anime-game-launcher-gtk-bin/an-anime-game-launcher-gtk-bin.pacscript:26]:
```suggestion
    sudo chmod +x "anime-game-launcher"
```

File: packages/an-anime-game-launcher-gtk-bin/an-anime-game-launcher-gtk-bin.pacscript [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    wget -q "https://aur.archlinux.org/cgit/aur.git/plain/icon.png?h=an-anime-game-launcher-gtk-bin" -O icon.png
+    sudo install -Dm644 icon.png "${STOWDIR}/${name}/usr/share/pixmaps/an-anime-game-launcher.png"
+
+    # Get desktop file and install it
+    wget -q "https://aur.archlinux.org/cgit/aur.git/plain/an-anime-game-launcher.desktop?h=an-anime-game-launcher-gtk-bin" -O an-anime-game-launcher.desktop
+    sudo install -Dm644 an-anime-game-launcher.desktop -t "${STOWDIR}/${name}/usr/share/applications"
+
+    sudo install -dm755 "${STOWDIR}/${name}/usr/lib/${pkgname}"
+    sudo install -dm755 "${STOWDIR}/${name}/usr/bin/"
+    sudo cp an-anime-game-launcher "${STOWDIR}/${name}/usr/lib/${pkgname}/"
```
Comment (Henryws) [inline packages/an-anime-game-launcher-gtk-bin/an-anime-game-launcher-gtk-bin.pacscript:44]:
```suggestion
    sudo cp anime-game-launcher "${STOWDIR}/${name}/usr/lib/${pkgname}/"
```

File: packages/an-anime-game-launcher-gtk-bin/an-anime-game-launcher-gtk-bin.pacscript [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+    sudo ln -s -r "${STOWDIR}/${name}/usr/lib/${pkgname}/an-anime-game-launcher" "${STOWDIR}/${name}/usr/bin/an-anime-game-launcher"
```
Comment (Henryws) [inline packages/an-anime-game-launcher-gtk-bin/an-anime-game-launcher-gtk-bin.pacscript:46]:
```suggestion
    sudo ln -s -r "${STOWDIR}/${name}/usr/lib/${pkgname}/anime-game-launcher" "${STOWDIR}/${name}/usr/bin/an-anime-game-launcher"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]

**Pull request merged**